example/sinatra/webapp.rb

```diff
@@ -0,0 +1,77 @@
+require 'veritrans'
+require 'json'
+require 'sinatra'
+
+Veritrans.setup do
+  config.load_yml File.dirname(__FILE__) + "/veritrans.yml#development"
+end
+
+# Alternative way to to initialize core api client:
+# mt_client = Midtrans.new(
+#   server_key: "your server key",
+#   client_key: "your client key",
+#   api_host: "https://api.sandbox.midtrans.com", # default
+#   http_options: {}, # optional
+#   logger: Logger.new(STDOUT), # optional
+#   file_logger: Logger.new(STDOUT), # optional
+# )
+
+set :public_folder, File.dirname(__FILE__)
+set :views, File.dirname(__FILE__)
+set :run, $0 == __FILE__
+
+def generate_order_id
+  "sinatra-example-#{Time.now.to_i}"
+end
+
+get "/" do
+  erb :index
+end
+
+get "/snap" do
+  result = Veritrans.create_widget_token(
```

Comment:
why still use `create_widget_token` ? use the proper function named `snap`. You are confusing merchant.